### ISSUE TITLE
False positive on vertical_parameter_alignment with default parameters in multiple lines

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1129]
This shouldn't trigger:

```swift
internal func regex(_ pattern: String,
                    options: NSRegularExpression.Options = [.anchorsMatchLines,
                                                            .dotMatchesLineSeparators]) -> NSRegularExpression 
```
